richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/parser/markdown/MarkdownImport.kt

```diff
@@ -330,14 +322,17 @@ internal fun ASTNode.orderedList(
         val listNumber = findChildOfType(MarkdownTokenTypes.LIST_NUMBER)?.getTextInNode(content)
         if (paragraph == null) {
             paragraph = RichParagraph(
-                type = RichParagraph.Type.OrderedList(listNumber.toString().toIntOrNull() ?: 0)
+                type = RichParagraph.Type.OrderedList(listNumber.toString().toIntOrNull() ?: 1)
             )
         }
-        paragraph!!.children.addAll(
-            parser.buildSpans(
-                content,
-                paragraph!!,
-                children.filter { it.type != MarkdownTokenTypes.EOL }
+        paragraph!!.children.add(
```

Comment:
I have added children of ordered list, as found in the markdown, but editor doesn't show new lines between them

Reply:
Considering the whole paragraph is a OrderedListItem / UnorderedListItem , This can be simplified to adding a single paragraph

**Mistake** : I thought items of a OrderedList Paragraph would become OrderedList items

Maybe we should also rename Paragraph OrderedList to OrderedListItem